src/LiveComponent/src/EventListener/DeferLiveComponentSubscriber.php

```diff
@@ -31,11 +31,22 @@ final class DeferLiveComponentSubscriber implements EventSubscriberInterface
     public function onPostMount(PostMountEvent $event): void
     {
         $data = $event->getData();
+
         if (\array_key_exists('defer', $data)) {
-            $event->addExtraMetadata('defer', true);
+            trigger_deprecation('symfony/ux-live-component', '2.17', 'The "defer" attribute is deprecated and will be removed in 3.0. Use the "loading" attribute instead.');
+            if ($data['defer']) {
+                $event->addExtraMetadata('loading', 'defer');
+            }
             unset($data['defer']);
         }
 
+        if (\array_key_exists('loading', $data)) {
+            if (\in_array($data['loading'], ['defer', 'lazy'], true)) {
```

Comment:
Ok, what if the user does `loading="foo"`? On the one hand, I think we should throw an exception. On the other hand, maybe we should allow that and render it as an attribute. Right now, I think we're doing neither - as it looks like `loading="foo"` would not be rendered due to the `unset()` below, right?

Reply:
The only HTML element that supports the `loading` attribute today are `img` and `iframe`, probably `portal` soon. 

None of them can be a LiveComponent so i think we can ignore this for now. And decide what to do if someone raises some concern about that ?

wdyt ?

Reply:
In that case, throwing an exception would be the most helpful for the user and the most conservative (if we removed the exception and allowed the attribute to render later, there is no way that will affect anyone).

Reply:
So we should ignore `false`  and `''` value, keep `lazy` or `defer`, and throw elsewhen.. right ?